package/package.json

```diff
@@ -77,7 +77,7 @@
     "path": "0.12.7",
     "react-native-markdown-package": "1.8.2",
     "react-native-url-polyfill": "^1.3.0",
-    "stream-chat": "^8.57.6",
+    "stream-chat": "9.0.0-rc.4",
```

Comment:
Just a reminder for my future self to update this whenever the new `stream-chat` version is released.